index.html

```diff
@@ -0,0 +1,201 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<meta charset="utf-8">
+	<meta http-equiv="X-UA-Compatible" content="IE=edge">
+	<title></title>
+	<link rel="stylesheet" href="quiz.css">
+</head>
+<body>
+  <article class="cards">
+    <header class="header">
+      <h1>InsertProductTitle</h1>
+    </header>
+    <div id="firstSection">
+    	<img src="woodBurningPic4.jpg" alt="Dragon">
```

Comment:
`<img>` tags take on properties of both inline and block level elements. You can satisfy the requirements by wrapping this tag in a `<div>`.